src/workloads/scale/BulkLoading.yml

```diff
@@ -0,0 +1,288 @@
+  # does not appear to be impacted by the batch size.
+  - &WriteOp { WriteCommand: insertOne, Document: *Document }
+  - &WriteOps10x [
+      *WriteOp, *WriteOp, *WriteOp, *WriteOp, *WriteOp,
+      *WriteOp, *WriteOp, *WriteOp, *WriteOp, *WriteOp,
+    ]
+  - &WriteOps100x {
+      ^FlattenOnce:
+        [
+          *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
+          *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
+        ],
+    }
+  - &UpsertOp {
+      WriteCommand: updateOne,
+      Filter: { sensorId: { ^RandomInt: { min: 0, max: 200000000 } } },
+      Update:
+        {
+          $set:
+            { eventId: 0, dataField1: { ^FastRandomString: { length: 10 } } },
+        },
+      OperationOptions: { Upsert: true },
+    }
+  - &UpsertOps10x [
+      *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp,
+      *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp,
+    ]
+  - &UpsertOps100x {
+      ^FlattenOnce:
+        [
+          *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x,
+          *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x,
+        ],
+    }
+
+ActorTemplates:
+  - TemplateName: BulkWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: 20 } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: *MeasurementDuration } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: bulkWrite
+                OperationCommand:
+                  WriteOperations:
+                    { ^Parameter: { Name: "Operations", Default: *WriteOps100x } }
+  - TemplateName: NormalWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: 30 } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: *MeasurementDuration } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: insertOne
+                OperationCommand:
+                  Document: *Document
+
+Actors:
+  # Each invocation of this actor bulk loads about 35 GB of data to populate the DB
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: WarmupLoadData
```

Comment:
hmm is it warmup or load? I would argue it is load (and this is what you have listed at the top of the file).